tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -289,6 +318,7 @@ def test_logs_surfacing_scheduler_basic_flow(self, cluster):
             headers=headers,
         )
 
+        # TODO [sb]: call cluster's /logs endpoint once introduced.
```

Comment:
actualy not sure we even need this part where we read the cluster logs on the server, shouldnt it be enough that things are flowing prperly if we get a positive response back from Den? (and same for the abovre in `test_status_scheduler_basic_flow`

Reply:
I think we actually need to get the logs from server.log, to verify that the last server logs were indeed sent to den (we check the content of the get logs request, not just that the request to den succeeded).